cairo-contracts/packages/core/src/commitment/types.cairo

```diff
@@ -155,3 +151,56 @@ pub impl StateRootZero of Zero<StateRoot> {
         self.root.len() > 0
     }
 }
+
+pub impl StoreU8Array of Store<Array<u8>> {
```

Comment:
Since this is for the root, and the root is used in hot paths, let’s add a few unit tests:
- Roundtrip (de)serialization
- Store some roots and check if reading the state matches the expected result